examples/golang/providers/custom-template/.gitignore

```diff
@@ -1,2 +1,3 @@
 keyvalue-inmemory
-keyvalue-inmemory.par.gz
\ No newline at end of file
+keyvalue-inmemory.par.gz
+build/*
```

Comment:
I actually chose to commit the component artifact so that users of the custom provider can just build the provider and automatically have a component to test with. I'm not strongly held to this methodology, but I do like the simplicity of it

Reply:
isnt it better to add an instruction to perform a wash build for the component ? Instead of placing the build results in git?

Reply:
I updated the instructions in Readme to build the component.